prac_05/wimbledon.py

```diff
@@ -0,0 +1,43 @@
+"""
+Wimbledon
+Estimate: 40 mins 10:54-10:57, 11:08
+Actual: 30 mins 11:34
+"""
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    winners = get_wimbledon_winners()
+
+    winner_to_occurrence = {}
```

Comment:
Maybe instead of setting the empty dictionary outside of the get_winner_to_occurrences function you could set it inside the function so you don't have to pass in the empty dictionary as an argument. This may make the function more reusable in other programs as well